lib/DxilPIXPasses/DxilAnnotateWithVirtualRegister.cpp

```diff
@@ -43,8 +43,10 @@
 
 uint32_t CountStructMembers(llvm::Type const *pType) {
   uint32_t Count = 0;
-
-  if (auto *ST = llvm::dyn_cast<llvm::StructType>(pType)) {
+  if (auto *VT = llvm::dyn_cast<llvm::VectorType>(pType)) {
+    Count = CountStructMembers(VT->getVectorElementType()) *
```

Comment:
Don't need to CountStructMembers on vector element type.
It must be scalar types.

Reply:
Thanks! Didn't know that..